Code/Tools/FontImporter/FontImporter.cpp

```diff
@@ -0,0 +1,382 @@
+#include <FontImporterPCH.h>
+#include <Foundation/IO/FileSystem/FileReader.h>
```

Comment:
A tip regarding #includes and the PCH file: Clang-format will rearrange include directives to sort them alphabetically. That's sometimes a problem, and always a problem with the PCH, because that one has to be the first. Add an empty line after the PCH include, to prevent clang-format from sorting it with the others.